common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.java

```diff
@@ -29,14 +29,63 @@
 
 import com.microsoft.identity.common.exception.ClientException;
 
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_DEBUG_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_RELEASE_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.BROKER_HOST_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.BROKER_HOST_APP_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.COMPANY_PORTAL_APP_RELEASE_SIGNATURE;
+
 /**
  * Represents packageName and SignatureHash of a broker app.
  */
 public class BrokerData {
+
+    public static final BrokerData MICROSOFT_AUTHENTICATOR_DEBUG = new BrokerData(
```

Comment:
>MICROSOFT_AUTHENTICATOR_DEBUG [](start = 35, length = 29)

Should these be part of a separate class.. 

public static final class BrokerApps
{
public BrokerData  Authenticator_Release {..};
public BrokerData CompanyPortalRelease {..}
..
..
}

Reply:
We could do it if we wanted to....I chose not to do it because it requires creating another class just for this tiny purpose